zkevm-circuits/src/util.rs

```diff
@@ -54,6 +54,7 @@ pub struct Challenges<T = Challenge> {
 impl Challenges {
     /// Construct `Challenges` by allocating challenges in specific phases.
     pub fn construct<F: FieldExt>(meta: &mut ConstraintSystem<F>) -> Self {
+        let _dummy_col = meta.advice_column();
```

Comment:
I think this still adds small overhead for both prover and verifier, it'd be nice if we could avoid this workaround (we just need to call `{KeccakBitConfig,KeccakPackedConfig}::configure(meta, challenge_exprs)` before `Challenges::construct(meta)`).

But I understand it's annoying for people to encounter the panic, so I'm also fine to keep it here.